gravitee-apim-gateway/gravitee-apim-gateway-handlers/gravitee-apim-gateway-handlers-api/src/main/java/io/gravitee/gateway/reactive/handlers/api/security/plan/SecurityPlan.java

```diff
@@ -81,15 +81,7 @@ public Single<Boolean> canExecute(HttpExecutionContext ctx) {
         return policy
             .extractSecurityToken(ctx)
             .doOnSuccess(securityToken -> ctx.setInternalAttribute(ATTR_INTERNAL_SECURITY_TOKEN, securityToken))
```

Comment:
We could gain an extra `.doOnSuccess()` by moving the `setInternalAttribute()` into the `flatMap()`.